ccd-config-generator/src/test/java/uk/gov/hmcts/reform/fpl/CCDConfig.java

```diff
@@ -44,6 +46,23 @@ public void configure() {
         caseField("submittedForm", "Attached PDF", "Document");
     }
 
+    private void buildWorkBasketResultFields() {
+        workBasketResultFields()
+            .field(CaseData::getCaseName, "Case name")
+            .field(CaseData::getFamilyManCaseNumber, "FamilyMan case number")
+            .field("[STATE]", "State")
```

Comment:
Should we hard code [STATE] in the generator so it's always generated without having to explicitly declare it? Would teams ever want to not have state do you think?

Reply:
We could always make it configurable later on if necessary

Reply:
if we assume all teams will use this, then yes it could be hardcoded. I wouldn't want any of the service teams forced into having it however

Reply:
I guess we could create some enums for meta data fields to make stuff like this super easy?

Reply:
We could have an opt out builder method; something like 'noState()'. I'm thinking that if we'd expect most teams to have state then it should be on by default, with an opt out.

Is [STATE] a special keyword in CCD config?

Reply:
Just checked Probate and they don't have STATE in their WBIFs, so let's keep your PR as it is.

Reply:
CCD config does some magic, metadata fields (same level as case data) are accessible via this notation "[keyword]", another example is ["JURISIDCTION"] or ["CASE_REFERENCE"]

Reply:
also, WBIF state is there by default

Reply:
I guess we could have builder methods for the known metadata fields, that might help new users with more discoverable config. Can always do that later on someday, not esp a priority.